packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButton.tsx

```diff
@@ -8,12 +8,15 @@ export type RadioButtonSize = 'sm' | 'md';
 export interface RadioButtonProps {
   size?: RadioButtonSize;
   disabled?: boolean;
+  name?: string;
   active: boolean;
-  onClick: () => void;
+  id: string;
+  onChange: () => void;
 }
 
 const getRadioButtonStyles = stylesFactory((theme: GrafanaTheme, size: RadioButtonSize) => {
-  const { padding, fontSize, height } = getPropertiesForButtonSize(theme, size);
+  const { fontSize, height } = getPropertiesForButtonSize(theme, size);
+  const horPadding = theme.spacing[size] ?? theme.spacing.md;
```

Comment:
what `hor` stands for?

Reply:
horizontal - should I just include the whole word in the variable to make it more clear?

Reply:
yeah, at first sight both me and @ryantxu we had problems getting this 🗡